src/callbacks/callbacks.jl

```diff
@@ -166,10 +166,14 @@ NVTX.@annotate function rrtmgp_model_callback!(integrator)
         )
         # multiply by 1000 to convert from kg/m^2 to g/m^2
         @. ᶜlwp =
-            1000 * Y.c.ρ * TD.liquid_specific_humidity(thermo_params, ᶜts) * ᶜΔz
+            1000 *
```

Comment:
Are we doing this unit conversion because RRTMG needs it in g/m^2? If so, it's better to replace 1000 by `kg_to_g_conversion` or something like it. Otherwise, sticking with SI units is better.

Reply:
Yes RRTMG needs g/m^2, I'll replace it.